backend/webserver/converters.py

```diff
@@ -125,7 +125,12 @@ def send_follow_request(self, request_data):
         return converted_data
     
     def send_post_inbox(self, post, request: HttpRequest):
+        
         host = get_host_from_absolute_url(post.author.get_url(request))
+        if post.source == "":
+            post.source = "www.default.com"
```

Comment:
you are changing the post object in OUR database. this is not the way to do it 👎